addons/sale_stock/views/account_invoice_views.xml

```diff
@@ -16,7 +16,7 @@
     <template id="report_invoice_document_inherit_sale_stock" inherit_id="account.report_invoice_document">
         <!-- HTML Preview for mobile responsive -->
         <xpath expr="//div[@name='reference']" position="after">
-            <div t-att-class="o_grid_layout" t-if="o.incoterms_id" groups="sale_stock.group_display_incoterm">
+            <div t-att-class="o_grid_layout" t-if="o.incoterms_id" groups="sale_stock.group_display_incoterm" invisible="1">
                 <strong>Incoterms:</strong>
                 <p t-field="o.incoterms_id.code"/>
```

Comment:
`"o.incoterm_id.code"`?

Reply:
I agree with your comment since it's not a duplicate, there is no other inheritance of account.report_invoice_document that adds the incoterm, if I haven't failed my greps.
But note that your commit 569a498d360b99b2a1d8e64a6e6bcc259d0910e4 does completely remove addons/sale_stock/views/account_invoice_views.xml ! So that means it should be added back in master too...

Reply:
Ok then